OpenRA.Game/Player.cs

```diff
@@ -244,28 +249,40 @@ public bool IsAlliedWith(Player p)
 			return RelationshipWith(p) == PlayerRelationship.Ally;
 		}
 
-		public static Color PlayerRelationshipColor(Actor a)
+		/// <summary>Returns <see cref="color"/>, ignoring player relationship colors.</summary>
+		public static Color GetColor(Player p) => p.color;
```

Comment:
Having `p.Color` return the colour with relationship colours and `Player.GetColor(p)` return the colour without relationship colours feels quite obtuse.

Can we please have a single method which takes an argument as to whether relationship colours should be used or ignored?

Reply:
It's not supposed to be used. It's there only for the lua interface and setting up the game state